tests/scipy_signal_test.py

```diff
@@ -91,7 +91,7 @@ def testConvolutions2D(self, xshape, yshape, dtype, mode, jsp_op, osp_op):
           jtu.format_shape_dtype_string(shape, dtype), axis, type, bp),
        "shape": shape, "dtype": dtype, "axis": axis, "type": type, "bp": bp}
       for shape in [(5,), (4, 5), (3, 4, 5)]
-      for dtype in default_dtypes
+      for dtype in jtu.dtypes.floating + jtu.dtypes.integer
```

Comment:
Any reason for this change? `default_dtypes` tests `all_floating` and `all_integer`, so this change results in less test coverage.

Reply:
Whoops, I missed this before the rebase/push, so let me get this fixed.

I'd like to exclude the complex types in `testDetrend`

Up at line 35 we have 
```python
default_dtypes = jtu.dtypes.floating + jtu.dtypes.integer + jtu.dtypes.complex
```
where I added the `jtu.dtypes.complex` part;  so I don't think that this module was testing `all_floating` or `all_integer` (at least how I understand it).

I could define a new `default_dtypes_plus_complex` and leave line 94 as `dtype in default_dtypes`; would that work?

Reply:
Oh I see - my mistake. I think this is fine as-is.